example-app/src/main/java/com/hcaptcha/example/MainActivity.java

```diff
@@ -112,7 +114,14 @@ public void onFailure(HCaptchaException e) {
                         Log.d(TAG, "hCaptcha failed: " + e.getMessage() + "(" + e.getStatusCode() + ")");
                         setErrorTextView(e.getMessage());
                     }
+                })
+                .addOnOpenListener(new OnOpenListener() {
+                    @Override
+                    public void onOpen() {
+                        Toast.makeText(MainActivity.this, "hCaptcha shown", Toast.LENGTH_SHORT).show();
```

Comment:
For some reasons this Toast does not show in the emulator Pixel XL API 30.

Reply:
Hmm, cannot reproduce this on my end (tried the same emulator)
<img width="250" alt="Screen Shot 2022-05-04 at 2 02 33 PM" src="https://user-images.githubusercontent.com/2169738/166677467-b5040e87-348e-458e-a648-fa23b59a0e42.png">

Also tried with most recent emulator (Tiramisu) and actual devices, toast shows consistently

Reply:
No worries. Should be something on my setup.